### ISSUE TITLE
When I use FFmpegFrameGrabber to collect local pictures and run the program to release grabber, grabber.release()  method throws an exception and crashes the virtual machine

### ISSUE THREAD

**Issue by eguid** [bytedeco/javacv#1453]
When I use FFmpegFrameGrabber to collect local pictures and run the program to release grabber, grabber.release() is in
```
// Close the video codec
         if (video_c != null) {
             avcodec_free_context(video_c);
             video_c = null;
         }
```
crashes the virtual machine,what should I do? Thank you!

crashes error content:
`---------------  T H R E A D  ---------------

Current thread (0x00000000035e3800):  JavaThread "main" [_thread_in_native, id=11360, stack(0x0000000003330000,0x0000000003430000)]

siginfo: ExceptionCode=0xc0000005, reading address 0x0000000026952078

Registers:
RAX=0x000000001f1f8700, RBX=0x000000000342f348, RCX=0x0000000026952080, RDX=0x0000000026952080
RSP=0x000000000342f2f0, RBP=0x000000000342f498, RSI=0x000000001cd0a3c0, RDI=0x000000000342f4b8
R8 =0x000000001cd6f1c0, R9 =0x0000000000000001, R10=0x0000000000000003, R11=0x000000000342f290
R12=0x0000000000000000, R13=0x00000000181cbcc8, R14=0x000000000342f4b8, R15=0x00000000035e3800
RIP=0x00007ffb52fb93cd, EFLAGS=0x0000000000010202

Top of Stack: (sp=0x000000000342f2f0)
0x000000000342f2f0:   000000000342f4b8 00007ffb52fb9cfc
0x000000000342f300:   000000001cd6cf00 000000001cd0a3c0
0x000000000342f310:   000000000342f4b8 00007ffb2de46e0b
0x000000000342f320:   000000001cd6cf00 00007ffb52fb93d6
0x000000000342f330:   00000000034dedb0 000000001cd6cf00
0x000000000342f340:   000000001f1f8700 000000001cd6f1c0
0x000000000342f350:   000000001cd6cf00 00007ffafd9bad6d
0x000000000342f360:   00000000035e3800 000000000342f4b8
0x000000000342f370:   0000000000000000 00000000036e83fd
0x000000000342f380:   0000000000000002 000000001cd6cf00
0x000000000342f390:   000000000342f408 00007ffafc954989
0x000000000342f3a0:   000000000342f948 000000001dc28050
0x000000000342f3b0:   00000000181cbcc8 0000000018060b90
0x000000000342f3c0:   000000000342f458 00000000035e39f8
0x000000000342f3d0:   0000000000000000 000000006a1f4bed
0x000000000342f3e0:   00000000035e39f8 000000000342f498 

Instructions: (pc=0x00007ffb52fb93cd)
0x00007ffb52fb93ad:   48 83 c4 28 c3 cc cc cc cc cc cc cc cc cc cc cc
0x00007ffb52fb93bd:   cc cc cc 48 83 ec 28 48 85 c9 74 0d 48 83 e1 f8
0x00007ffb52fb93cd:   48 8b 49 f8 e8 0a 09 00 00 48 83 c4 28 c3 cc cc
0x00007ffb52fb93dd:   cc cc cc cc cc cc cc cc cc cc cc cc cc cc cc cc 

Register to memory mapping:

RAX=0x000000001f1f8700 is an unknown value
RBX=0x000000000342f348 is pointing into the stack for thread: 0x00000000035e3800
RCX=0x0000000026952080 is an unknown value
RDX=0x0000000026952080 is an unknown value
RSP=0x000000000342f2f0 is pointing into the stack for thread: 0x00000000035e3800
RBP=0x000000000342f498 is pointing into the stack for thread: 0x00000000035e3800
RSI=0x000000001cd0a3c0 is an unknown value
RDI=0x000000000342f4b8 is pointing into the stack for thread: 0x00000000035e3800
R8 =0x000000001cd6f1c0 is an unknown value
R9 =0x0000000000000001 is an unknown value
R10=0x0000000000000003 is an unknown value
R11=0x000000000342f290 is pointing into the stack for thread: 0x00000000035e3800
R12=0x0000000000000000 is an unknown value
R13=0x00000000181cbcc8 is pointing into metadata
R14=0x000000000342f4b8 is pointing into the stack for thread: 0x00000000035e3800
R15=0x00000000035e3800 is a thread

Stack: [0x0000000003330000,0x0000000003430000],  sp=0x000000000342f2f0,  free space=1020k
Native frames: (J=compiled Java code, j=interpreted, Vv=VM code, C=native code)
C  [msvcrt.dll+0x193cd]
C  [avutil-56.dll+0x6e0b]
C  [avcodec-58.dll+0x152ad6d]
C  [avcodec-58.dll+0x4c4989]
C  [jniavcodec.dll+0x34bed]
C  0x000000000370100c

Java frames: (J=compiled Java code, j=interpreted, Vv=VM code)
j  org.bytedeco.javacpp.avcodec.avcodec_free_context(Lorg/bytedeco/javacpp/avcodec$AVCodecContext;)V+0
j  org.bytedeco.javacv.FFmpegFrameGrabber.releaseUnsafe()V+128
j  org.bytedeco.javacv.FFmpegFrameGrabber.release()V+1
j  org.bytedeco.javacv.FFmpegFrameGrabber.stop()V+1
j  cc.eguid.cv.videoRecorder.recorder.ImgPushMediaStream.close()V+57
j  cc.eguid.cv.videoRecorder.recorder.ImgPushMediaStream.go(J)V+225
j  cc.eguid.cv.videoRecorder.recorder.ImgPushMediaStream.main([Ljava/lang/String;)V+27
v  ~StubRoutines::call_stub

---------------  P R O C E S S  ---------------

Java Threads: ( => current thread )
  0x000000001c173800 JavaThread "AWT-Windows" daemon [_thread_in_native, id=12572, stack(0x00000000215d0000,0x00000000216d0000)]
  0x000000001dc29800 JavaThread "Java2D Disposer" daemon [_thread_blocked, id=3932, stack(0x000000001f5f0000,0x000000001f6f0000)]
  0x000000001b7d3800 JavaThread "JavaCPP Deallocator" daemon [_thread_blocked, id=11928, stack(0x000000001cd80000,0x000000001ce80000)]
  0x000000001a77d800 JavaThread "Service Thread" daemon [_thread_blocked, id=12052, stack(0x000000001b2b0000,0x000000001b3b0000)]
  0x000000001a786000 JavaThread "C1 CompilerThread3" daemon [_thread_blocked, id=11676, stack(0x000000001b1b0000,0x000000001b2b0000)]
  0x000000001a785800 JavaThread "C2 CompilerThread2" daemon [_thread_blocked, id=6208, stack(0x000000001b0b0000,0x000000001b1b0000)]
  0x000000001a782800 JavaThread "C2 CompilerThread1" daemon [_thread_blocked, id=12360, stack(0x000000001afb0000,0x000000001b0b0000)]
  0x000000001a782000 JavaThread "C2 CompilerThread0" daemon [_thread_blocked, id=1540, stack(0x000000001aeb0000,0x000000001afb0000)]
  0x000000001a559000 JavaThread "JDWP Command Reader" daemon [_thread_in_native, id=5936, stack(0x000000001adb0000,0x000000001aeb0000)]
  0x000000001a556000 JavaThread "JDWP Event Helper Thread" daemon [_thread_blocked, id=14412, stack(0x000000001acb0000,0x000000001adb0000)]
  0x000000001a550800 JavaThread "JDWP Transport Listener: dt_socket" daemon [_thread_blocked, id=13604, stack(0x000000001abb0000,0x000000001acb0000)]
  0x000000001a53e800 JavaThread "Attach Listener" daemon [_thread_blocked, id=12020, stack(0x000000001aab0000,0x000000001abb0000)]
  0x000000001a53d000 JavaThread "Signal Dispatcher" daemon [_thread_blocked, id=6368, stack(0x000000001a9b0000,0x000000001aab0000)]
  0x000000001a4d0800 JavaThread "Finalizer" daemon [_thread_blocked, id=13856, stack(0x000000001a8b0000,0x000000001a9b0000)]
  0x00000000185dd800 JavaThread "Reference Handler" daemon [_thread_blocked, id=14220, stack(0x000000001a3b0000,0x000000001a4b0000)]
=>0x00000000035e3800 JavaThread "main" [_thread_in_native, id=11360, stack(0x0000000003330000,0x0000000003430000)]
`

**Comment by saudet** [comment]
It might be something that was fixed with FFmpeg 4.3.
Please try again with JavaCV 1.5.4-SNAPSHOT: http://bytedeco.org/builds/

**Comment by saudet** [comment]
I'm assuming that JavaCV 1.5.4 and/or FFmpeg 4.3.1 fixes this, but please let me know if you still have this issue! Thanks
